pallas-network/tests/protocols.rs

```diff
@@ -681,10 +681,10 @@ pub async fn local_state_query_server_and_client_happy_path() {
             let transaction_id_2 = Hash::from(txbytes);
             let index_2 = AnyUInt::MajorByte(0);
             let lovelace = AnyUInt::U32(1_792_960);
-            let hex_datum = "D8799FD8799F1AE06755BBFF1B00000193B36BC9F0FF";
-            let datum = hex::decode(hex_datum).unwrap().into();
-            let tag = TagWrap::<_, 24>::new(datum);
-            let inline_datum = Some((1_u16, tag));
+            //let hex_datum = "D8799FD8799F1AE06755BBFF1B00000193B36BC9F0FF";
+            //let datum = hex::decode(hex_datum).unwrap().into();
+            //let tag = TagWrap::<_, 24>::new(datum);
+            let inline_datum = None;
             let values_2 = localstate::queries_v16::TransactionOutput::Current(
```

Comment:
Sorry, I meant these lines (the offending query)

Reply:
Done in 424c816